haiku-apps/projectconceptor/projectconceptor-1.0.0.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A Haiku diagramming application like Microsoft Visio"
+DESCRIPTION="ProjectConceptor is a modular application for Haiku and BeOS \
+designed to view, edit and process information which can be represented by a \
+graph structure."
+APP="ProjectConceptor"
+HOMEPAGE="https://github.com/Paradoxianer/$APP"
+COPYRIGHT="2017 Paradoxianer"
```

Comment:
There're [a few more][0] copyright holders

- [Lee Thomason][1]
- [Be Incorporated][2]
- (could be a holder) [Al.V. Sarikov][3]

[0]: https://github.com/Paradoxianer/ProjectConceptor/search?utf8=%E2%9C%93&q=copyright&type=
[1]: https://github.com/Paradoxianer/ProjectConceptor/blob/dfef4b25fbd3dc620e144fef66a0879c4442a8f1/src/app/Tools/tinyxml/tinyxmlparser.cpp#L3
[2]: https://github.com/Paradoxianer/ProjectConceptor/blob/dfef4b25fbd3dc620e144fef66a0879c4442a8f1/src/app/Tools/ColorTools.h#L4
[3]: https://github.com/Paradoxianer/ProjectConceptor/blob/dfef4b25fbd3dc620e144fef66a0879c4442a8f1/src/app/ToolBar/CalendarToolItem/MonthWindow.cpp#L2